sale_inherit/models/sale_order.py

```diff
@@ -0,0 +1,20 @@
+from odoo import models, fields
```

Comment:
```suggestion
from odoo import fields, models
```

This should be organized in alphabetical order.